### ISSUE TITLE
Incompatibility with Python 3.11

### ISSUE THREAD

**Issue by beenje** [PierreRaybaut/guidata#56]
`guidata` can't be installed with Python 3.11

```
$ pip install -e .
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [14 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "guidata/setup.py", line 32, in <module>
          from guidata.utils import get_subpackages, get_package_data
        File "guidata/guidata/__init__.py", line 542, in <module>
          import guidata.config
        File "guidata/guidata/config.py", line 21, in <module>
          _ = get_translation("guidata")
              ^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "guidata/guidata/configtools.py", line 53, in get_translation
          _trans = gettext.translation(
                   ^^^^^^^^^^^^^^^^^^^^
      TypeError: translation() got an unexpected keyword argument 'codeset'
```

See https://docs.python.org/3/library/gettext.html

Changed in version 3.11: codeset parameter is removed.

**Comment by PierreRaybaut** [comment]
Forgot to update this issue. Sorry for this.
Version 2.3.1 fixes compatibility issues with Python 3.11:
https://github.com/PierreRaybaut/guidata/releases/tag/v2.3.1

Thanks for the feedback.

**Comment by beenje** [comment]
No pb. I had noticed :-)
I could also have mentioned it here. Thanks!
